### ISSUE TITLE
Update windows instructions in the wiki

### ISSUE THREAD

**Issue by vielm** [MaskRay/ccls#606]
Hey, I just set up ccls in Windows (I'm using it with emacs), and I ran into issues with getting ccls up and running by following the instructions on the wiki. I was just wondering if you would like me to update the Windows instructions (under "Build") with the steps that worked for me. 

I went through MSYS2 because it was faster (didn't have to compile clang), and one of the main differences was that to run ccls outside of MinGW I had to copy over all DLL's in MinGW's bin folder into the directory with the ccls excecutable. Without this ccls would constantly crash on start, probably because it was missing some of those libraries.

PS: Sorry if this is not where I should be bringing up this kind of thing, I'm very new to GitHub and I'm still learning how things work

PPS: Thanks for all the work in ccls, I've been using it the past few days and its awesome!

**Comment by ashrasmun** [comment]
I went through the same process. To be more precise, during installation I had to use the `--disable-timeout` option in order to download all the required packages. After the build, I had to copy `zlib1.dll`, `libwinpthread-1.dll`, `libstdc++-6.dll` and `libgcc_s_seh-1.dll` libraries from the `msys64\mingw64\bin` directory.

I can run `ccls` now, I think... all that happens is a console being opened, but that outcome is way more promising than having errors and crashes :)
